lambdatimes/src/components/Header.js

```diff
@@ -1,16 +1,60 @@
 import React from 'react';
+import styled from 'styled-components';
 
-// Refactor this component to use styled components and not classNames. 
+// Refactor this component to use styled components and not classNames.
 // You can find the corresponding CSS in the CSS/index.css file
```

Comment:
My advice would be to separate your styled components and import them into your presentational components to reduce clutter